Paintroid/src/androidTest/java/org/catrobat/paintroid/test/utils/RecyclerViewMatcher.kt

```diff
@@ -0,0 +1,58 @@
+package org.catrobat.paintroid.test.utils
+
+import android.content.res.Resources
+import android.content.res.Resources.NotFoundException
+import android.view.View
+import androidx.recyclerview.widget.RecyclerView
+import org.hamcrest.Description
+import org.hamcrest.Matcher
+import org.hamcrest.TypeSafeMatcher
+
+class RecyclerViewMatcher(private val recyclerViewId: Int) {
+
+    fun atPositionOnView(position: Int, targetViewId: Int): Matcher<View> {
+        return object : TypeSafeMatcher<View>() {
+            var resources: Resources? = null
+            var childView: View? = null
+
+            @SuppressWarnings("SwallowedException")
+            override fun describeTo(description: Description) {
+                var idDescription = recyclerViewId.toString()
+                if (resources != null) {
+                    idDescription = try {
+                        resources!!.getResourceName(recyclerViewId)
+                    } catch (var4: NotFoundException) {
+                        String.format(
```

Comment:
same as above, I think it's okay for a test helper method to throw an exception as it helps to show that something went wrong

Reply:
Same answer as above, is also a checked Exception